src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -274,4 +275,9 @@
     </None>
   </ItemGroup>
 
+  <!-- Configurations for Azure AD Kerberos. -->
+  <ItemGroup Condition="$([MSBuild]::IsOsPlatform('Windows'))">
```

Comment:
Not really needed as we only build on Windows. I will fix this for you.

Reply:
The AAD Kerberos feature can be supported in all platform/OS excepts the Ticket Cache support feature which only has dependency on Windows. 
- Removed this condition.
- Changed all Windows OS dependent files so that they are not included in build time using (iOS || MAC || ANDROID) condition.